NEWS.md

```diff
@@ -1,5 +1,6 @@
 # admiral 0.8.3
 
+- Second attempto to address issue where CRAN identified a failing test when "a strict Latin-1* locale" is used. (#1469)
```

Comment:
We don't use full stops in the rest of the `NEWS` so should be consistent with that.

```suggestion
- Second attempt to address issue where CRAN identified a failing test when "a strict Latin-1* locale" is used (#1469)
```

Reply:
apologies just rolling out of bed